### PR TITLE
✨ Provide changelog modal

### PR DESCRIPTION
**Pull request by superbuggy**

### CODE REVIEW

File: public/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# OSIM Changelog
```
Comment (JakubFrejlach) [inline public/CHANGELOG.md:1]: Does this mean we now need to keep updating both `CHANGELOG.md` and `public/CHANGELOG.md`? If so, is there way how to either keep just this one or somehow provide a "link" to the one in a root? I mean, having two CHANGELOG files calls for inconsistencies.

File: src/components/ChangeLog.vue [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            @click="closeModal"
+          ></button>
+        </div> 
+  
+      </template>
+      <template #body>
+        <div v-html="html.body" />
+      </template>
+      <template #footer>
+        <button class="btn btn-primary" @click="closeModal">Close</button>
```
Comment (JakubFrejlach) [inline src/components/ChangeLog.vue:54]: Isn't it redundant to have two differently looking closing buttons on one modal?

**Review by JakubFrejlach — APPROVED** [review]
Added suggestions and questions.

File: src/components/ChangeLog.vue (hunk 2 above)
Comment (superbuggy) [reply src/components/ChangeLog.vue:54]: It is, but one of them you have to scroll to see and is more obvious if you miss the light grey x in the upper left corner

File: public/CHANGELOG.md (hunk 1 above)
Comment (superbuggy) [reply public/CHANGELOG.md:1]: This is only in public for local development purposes--I change the content of the file to reflect that

Comment (superbuggy) [reply public/CHANGELOG.md:1]: Actually a symlink might be better here...

**Comment by tboyetteredhat** [comment]
@superbuggy I think there's some ui issues. 
image

- Close icon is not showing right.
- Scroll behavior on modal
- - Modal header
- - Close button

**Comment by C-Valen** [comment]
> @superbuggy I think there's some ui issues. image
> 
>     * Close icon is not showing right.
> 
>     * Scroll behavior on modal
> 
>     * * Modal header
> 
>     * * Close button

I thought the same as @tboyetteredhat.
Just pointing couple of suggestions:
- Move close icon top-right with `sticky`/`absolute` positioning as only closing option (being always visible).
- Add a bit of padding on body content

Overall the approach looks nice to me, considering the difficulties on having access to the file.

**Review by C-Valen — APPROVED** [review]
Nice one! Looks good 🔥 

One small consideration:
Isn't it more logic having the link on the footer, than in settings?

**Comment by superbuggy** [comment]
> Nice one! Looks good 🔥
> 
> One small consideration: Isn't it more logic having the link on the footer, than in settings?

That had been my thought but then someone pointed out (@d10n, maybe) it would be too easy to miss and/or inconvenient? We could include it in both places I think

**Review by d10n — APPROVED** [review]

**Pull request merged**